### PR TITLE
Use github actions instead of travis

### PR DESCRIPTION
**Pull request by Sammers21**
The global decision is to migrate from Travis to Github Actions.

### CODE REVIEW

File: .github/workflows/maven.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 11
+      uses: actions/setup-java@v1
+      with:
+        java-version: 11.0.x
+    - name: Build
+      run: mvn clean install -B -Pqulice --file pom.xml
```
Comment (amihaiemil) [inline .github/workflows/maven.yml]: @Sammers21 What is the ``-B`` parameter, what does it do? And is it necessary to specify ``--file pom.xml``? Maven will automatically search for it in the dir where it runs.

File: .github/workflows/maven.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 11
+      uses: actions/setup-java@v1
+      with:
+        java-version: 11.0.x
+    - name: Build
```
Comment (amihaiemil) [inline .github/workflows/maven.yml]: @Sammers21 Which name will be applied? This ``Build`` or the gobal ``Maven Action``?

File: .github/workflows/maven.yml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+name: Maven Action
+on: [push, pull_request]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 11
+      uses: actions/setup-java@v1
```
Comment (amihaiemil) [inline .github/workflows/maven.yml]: @Sammers21 Are these actions ours, or inherited from somewhere else?

**Comment by amihaiemil** [comment]
@g4s8 assign me here as well pls

**Comment by g4s8** [comment]
@0crat assign @amihaiemil

**Comment by 0crat** [comment]
> @0crat assign @amihaiemil  ([here](https://github.com/artipie/pypi-adapter/issues/19#issuecomment-584094046))

@g4s8 This pull request [#19](https://github.com/artipie/pypi-adapter/issues/19) is assigned to @amihaiemil[/z](https://www.0crat.com/u/amihaiemil), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/97ccef78-febd-48cd-ada7-1e9d0291a9db); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

**Comment by amihaiemil** [comment]
@Sammers21 Also, please, add a description to the PR and a link to the Issue that it solves.

File: .github/workflows/maven.yml (hunk 3 above)
Comment (Sammers21) [reply .github/workflows/maven.yml]: https://github.com/actions/setup-java

File: .github/workflows/maven.yml (hunk 2 above)
Comment (Sammers21) [reply .github/workflows/maven.yml]: what dou you mean

File: .github/workflows/maven.yml (hunk 1 above)
Comment (Sammers21) [reply .github/workflows/maven.yml]:
```
-B,--batch-mode                        Run in non-interactive (batch)
                                        mode (disables output color)
```

Comment (Sammers21) [reply .github/workflows/maven.yml]: > And is it necessary to specify --file pom.xml

Just for explicity

File: .github/workflows/maven.yml (hunk 2 above)
Comment (amihaiemil) [reply .github/workflows/maven.yml]: @Sammers21 I assume, when this Action will run, some name will be displayed somewhere, right? Which name will be displayed, "Build" or "Maven Action"?

**Comment by Sammers21** [comment]
> @Sammers21 Also, please, add a description to the PR and a link to the Issue that it solves.

I would not do it. The global decision is to migrate from Travis to Github Actions.

**Comment by amihaiemil** [comment]
@Sammers21 I fixed the description for you. Whenever you explain something to me, that explanation should actually be in the PR'd description :)

File: .github/workflows/maven.yml (hunk 2 above)
Comment (Sammers21) [reply .github/workflows/maven.yml]: The name of the step is `Build`

**Comment by 0crat** [comment]
Manual assignment of issues is discouraged, see [§19](http://www.zerocracy.com/policy.html#19): -5 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by 0crat** [comment]
Order was finished: +20 point(s) just awarded to @amihaiemil[/z](https://www.0crat.com/u/amihaiemil)

**Comment by 0crat** [comment]
Job was finished in 18 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Pull request merged**